intro-HPC/ch_web_portal.tex

```diff
@@ -0,0 +1,290 @@
+\chapter{Using the HPC-UGent web portal}
+\label{ch:web_portal}
+
+\href{https://openondemand.org/}{\strong{Open OnDemand}} (or \texttt{OoD} for short)
+is a web portal that provides ``one stop shop'' for accessing High Performance Computing resources.
+
+With Open OnDemand, you can upload and download files, create, edit, submit, and monitor jobs,
+run GUI applications, and connect via SSH, all via a standard web browser like Firefox, Chrome
+or Safari. You do not need to install or configure any client software,
+and no SSH key is required to connect to your VSC account via this web portal.
+
+To connect to the HPC-UGent infrastructure via Open OnDemand, visit:
+
+\begin{center}\url{https://login.hpc.ugent.be}\end{center}
+
+Note that you may only see a ``\emph{Submitting...}'' message appear for a couple of seconds,
```

Comment:
hmm, what was this again. i might be able to resolve this or make it more descriptive

Reply:
Is this a blocker for merging this PR?

It's easy to remove afterwards if it's fixed or made more descriptive...